hubstorage/__init__.py

```diff
@@ -5,3 +5,12 @@
 
 from .client import HubstorageClient, __version__
 from .batchuploader import ValueTooLarge
+import warnings
+
+
+DEPRECATION_MSG = """
+python-hubstorage is deprecated, please use python-scrapinghub
+>= 1.9.0 instead (https://pypi.python.org/pypi/scrapinghub).
+"""
```

Comment:
```
DEPRECATION_MSG = (
    'python-hubstorage is deprecated, please use python-scrapinghub '
    '>= 1.9.0 instead (https://pypi.python.org/pypi/scrapinghub).'
)
```

Let's make it a single line, multiline warning used on import with package name and version on different lines looks creepy

<img width="1202" alt="job_31_ _noop_ _scrapinghub" src="https://cloud.githubusercontent.com/assets/744331/20059462/4c1375aa-a4ee-11e6-8cbd-d836858824cb.png">